src/pages/workspace/expensifyCard/WorkspaceExpensifyCardDetailsPage.tsx

```diff
@@ -59,6 +61,14 @@ function WorkspaceExpensifyCardDetailsPage({route}: WorkspaceExpensifyCardDetail
     const displayName = PersonalDetailsUtils.getDisplayNameOrDefault(cardholder);
     const translationForLimitType = CardUtils.getTranslationKeyForLimitType(card.nameValuePairs?.limitType);
 
+    const deactivateCard = () => {
+        setIsDeactivateModalVisible(false);
+
+        // TODO: add API call when it's supported https://github.com/Expensify/Expensify/issues/407841
+
+        Navigation.goBack();
+    };
+
```

Comment:
When deactivating card, the Apps will show empty page (ref: https://github.com/Expensify/App/issues/53560) we need to close thí page before remove card from Onyx